src/network/networkd-json.c

```diff
@@ -921,6 +921,35 @@ static int captive_portal_append_json(Link *link, JsonVariant **v) {
         return json_variant_merge_objectb(v, JSON_BUILD_OBJECT(JSON_BUILD_PAIR_STRING("CaptivePortal", captive_portal)));
 }
 
+static int pref64_append_json(Link *link, JsonVariant **v) {
+        _cleanup_(json_variant_unrefp) JsonVariant *array = NULL, *w = NULL;
+        NDiscPREF64 *i;
+        int r;
+
+        assert(link);
+        assert(v);
+
+        if (!link->network || !link->network->ipv6_accept_ra_use_pref64)
+                return 0;
+
+        SET_FOREACH(i, link->ndisc_pref64) {
+                r = json_build(&array, JSON_BUILD_OBJECT(
+                                               JSON_BUILD_PAIR_IN6_ADDR_NON_NULL("Prefix", &i->prefix),
+                                               JSON_BUILD_PAIR_UNSIGNED("PrefixLength", i->prefix_len),
+                                               JSON_BUILD_PAIR_FINITE_USEC("Lifetime", i->lifetime_usec),
+                                               JSON_BUILD_PAIR_STRING("ConfigSource", network_config_source_to_string(NETWORK_CONFIG_SOURCE_NDISC)),
+                                               JSON_BUILD_PAIR_IN6_ADDR_NON_NULL("ConfigProvider", &i->router)));
```

Comment:
Maybe, more specifically, `"Router"` is better in this case?

BTW, for future extension, I think it is better to store config per-router.
```
NDisc -> Router1 -> Address (router address)
                 -> PREF64 -> Prefix, PrefixLength, LifetimeUSec,
                 -> Other options
      -> Router2 -> ...
```

Reply:
Route ?

Reply:
If that's strange, feel free to use anything you think better.